### ISSUE TITLE
how to get to PSM contract from repl / deploy script?

### ISSUE THREAD

**Issue by dckc** [Agoric/Agoric#6490]
## Describe the bug

 @thisispalash is working on #5822 and he asked me how his code can talk to the PSM contract. I'm struggling to find a way.

## To Reproduce

1. check out the community-dev branch
2. agoric init; agoric start; agoric open --repl
3. try to look up the PSM and amm
4. find amm but no PSM:

```
command[16] E(home.agoricNames).lookup('instance', 'psm')
history[16] unresolved Promise
command[17] E(home.agoricNames).lookup('instance', 'amm')
history[17] [Object Alleged: InstanceHandle]{}
```

It looks like `decentral-demo-config.json` doesn't start the PSM.

## Expected behavior

PSM contract started in an environment where  @thisispalash can get at it.

## Platform Environment
 - what OS are you using? what version of Node.js?
 - is there anything special/unusual about your platform?
 - what version of the Agoric-SDK are you using? (run `git describe --tags --always`)

```
$ git describe --tags --always
agoricxnet-7-412-g3482e0d98
```

## Additional context
community-dev branch

**Comment by dckc** [comment]
#6498 might help a lot

**Comment by Tartuffo** [comment]
This may get at least partially fixed by #6498
